reducers/scikit_feature_selectors.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import sys
+import os
+sys.path.insert(0, os.path.abspath('..'))
+
+import numpy
+from modules.toolbox import framework_tools as ft
+import setup
+from sklearn.feature_selection import VarianceThreshold
+from sklearn.feature_selection import SelectPercentile
+from sklearn.feature_selection import SelectKBest
+
+
+#http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.VarianceThreshold.html
+def VarianceThreshold_selector(data,target):
+    x_train, x_test, y_train, y_test = ft.get_train_test(data, target)
+
+    #Select Model
+    selector = VarianceThreshold(0) #Defaults to 0.0, e.g. only remove features with the same value in all samples
+
+    #Fit the Model
+    selector.fit(x_train, y_train)
+    data.drop(target, 1, inplace=True) #Remove target feature
+    features = selector.get_support(indices = True) #returns an array of integers corresponding to nonremoved features
```

Comment:
@Zak7062 One thing to keep in mind here is that there seems to be some duplicate code between this function and the next two. If at all possible when writing code, it's best to reduce duplication and separate out functionality that can be separated into a new function.